docfx/toc_entry.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_DOCFX_TOC_ENTRY_H
+#define GOOGLE_CLOUD_CPP_DOCFX_TOC_ENTRY_H
+
+#include <iosfwd>
+#include <string>
+
+/// An entry in table of contents
+struct TocEntry {
```

Comment:
We should probably be putting all of this into a `docfx` namespace.

Reply:
I agree.